modules/op-release-notes-1-14.adoc

```diff
@@ -314,3 +314,15 @@ With this update, {pipelines-title} General Availability (GA) 1.14.2 is availabl
 * Before this update, when you started a pipeline run using {pac}, {tekton-results} did not store information about this pipeline run. Because of this issue, the web console plugin did not include the pipeline run in the execution statistics display. With this update, {tekton-results} stores information about {pac} pipeline runs and these pipeline runs are included in the execution statistics display.
 
 * Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher component failed to process some of the pending pipeline runs and they remained in a pending state. With this update, {pac} pipeline runs are processed correctly.
+
+[id="release-notes-1-14-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.14.3
+
+With this update, {pipelines-title} General Availability (GA) 1.14.2 is available on {OCP} 4.12 and later versions.
+
+[id="fixed-issues-1-14-3_{context}"]
+=== Fixed issues
+
+* Before this update, when you started many pipeline runs using {pac} at the same time and these pipelines runs included a `max-keep-run` annotation, the {pac} watcher executed the pipeline runs in the wrong order and some pipeline runs could remain stuck in a pending state for a long time. With this update, the pipeline runs are executed in the proper order and without delay.
+
+* Before this update, when you used the `tkn pr logs -f` command to view the logs for a running pipeline, the command line utility stopped responding, even if the pipeline run completyed successfully. With  this update, the `tkn pr logs -f` command properly displays the log information and exits.
```

Comment:
This is not cherry-picked so please remove

Reply:
@piyush-garg we are shipping this right with the latest release ?

Reply:
Not removing until there is clarity on the matter

Reply:
yes as I informed you on slack, we dont need to remove it now

Reply:
s/completyed/completed